src/main/java/gov/usgs/owi/nldi/controllers/NetworkController.java

```diff
@@ -48,15 +51,22 @@ public NetworkController(LookupDao inLookupDao, StreamingDao inStreamingDao,
 	@GetMapping(value="linked-data/comid/{comid}/navigate/{navigationMode}", produces= MediaType.APPLICATION_JSON_VALUE)
 
 	@Deprecated
-	public void getFlowlines(HttpServletRequest request, HttpServletResponse response,
-			@PathVariable(Parameters.COMID) @Range(min=1, max=Integer.MAX_VALUE) String comid,
-			@PathVariable(Parameters.NAVIGATION_MODE) @Pattern(regexp=REGEX_NAVIGATION_MODE) String navigationMode,
-			@RequestParam(value=Parameters.STOP_COMID, required=false) @Range(min=1, max=Integer.MAX_VALUE) String stopComid,
-			@Parameter(description=Parameters.DISTANCE_DESCRIPTION)
-				@RequestParam(value=Parameters.DISTANCE, required=false, defaultValue=Parameters.MAX_DISTANCE)
-			@Pattern(message=Parameters.DISTANCE_VALIDATION_MESSAGE, regexp=Parameters.DISTANCE_VALIDATION_REGEX) String distance,
-			@RequestParam(value=Parameters.LEGACY, required=false) String legacy) throws Exception {
+	public void getFlowlines(
+		HttpServletRequest request, HttpServletResponse response,
+		@PathVariable(Parameters.COMID) @Range(min=1, max=Integer.MAX_VALUE) String comid,
+		@PathVariable(Parameters.NAVIGATION_MODE) @Pattern(regexp=REGEX_NAVIGATION_MODE) String navigationMode,
+		@RequestParam(value=Parameters.STOP_COMID, required=false) @Range(min=1, max=Integer.MAX_VALUE) String stopComid,
+		@Parameter(description=Parameters.DISTANCE_DESCRIPTION)
+		@RequestParam(value=Parameters.DISTANCE, required=false, defaultValue=Parameters.MAX_DISTANCE)
+		@Pattern(message=Parameters.DISTANCE_VALIDATION_MESSAGE, regexp=Parameters.DISTANCE_VALIDATION_REGEX) String distance,
+		@RequestParam(value=Parameters.LEGACY, required=false) String legacy) throws Exception {
+
 		BigInteger logId = logService.logRequest(request);
+		if (stopComid != null && (Integer.parseInt(stopComid) < Integer.parseInt(comid))) {
+			logService.logRequestComplete(logId, response.getStatus());
```

Comment:
There is an issue here because the response status is still OK at this point.  Probably should use BAD_REQUEST or see if there is some other way to do that.  There must not be a test for this if this is currently passing.

Reply:
I think all the tests were written just to check that logRequestComplete() is invoked, not to check the status.  I'll try updated these and checking for 400.